packages/mux-player/src/types.d.ts

```diff
@@ -35,6 +35,7 @@ export type MuxTemplateProps = Partial<MuxPlayerProps> & {
   metadataVideoTitle: string;
   metadataViewerUserId: string;
   noHotKeys: boolean;
+  placeholder: string;
```

Comment:
this is the one place `placeholder` wasn't next to `poster`. That's because `MuxTemplateProps` inherits from `MuxPlayerProps` which in turn inherits from `MuxVideoElement` which already supplies us with `poster`. So in a way, yeah. `placeholder` is once again next to `poster`.